applications/nrf5340_audio/src/audio/streamctrl_unicast_server.c

```diff
@@ -658,17 +485,14 @@ int streamctrl_start(void)
 	ret = audio_system_init();
 	ERR_CHK(ret);
 
-	ret = data_fifo_init(&ble_fifo_rx);
-	ERR_CHK_MSG(ret, "Failed to set up ble_rx FIFO");
-
 	ret = zbus_subscribers_create();
 	ERR_CHK_MSG(ret, "Failed to create zbus subscriber threads");
 
 	ret = zbus_link_producers_observers();
 	ERR_CHK_MSG(ret, "Failed to link zbus producers and observers");
 
-	ret = audio_datapath_thread_create();
-	ERR_CHK_MSG(ret, "Failed to create audio datapath thread");
+	ret = le_audio_rx_init();
+	ERR_CHK_MSG(ret, "Failed to initialize rx path");
```

Comment:
```suggestion
	ERR_CHK_MSG(ret, "Failed to initialize RX path");
```
In streamctrl_unicast_client.c we only do "ERR_CHK", maybe we should have consistency for error checking in the streamctrl_start functions

Reply:
Perhaps, but that could in such case be another PR.